root/layout/header/bottom.tt

```diff
@@ -44,7 +47,7 @@
         </ul>
     </li>
     <li class="products">
-        <a href="[% doc_link('Products') %]">[%  l('Products') %]</a>
+        <span class="menu-header">[% l('Products') %]</span>
```

Comment:
This is pretty much "here is a list of all the products linked below" so I don't think it's worth keeping in any way in the menu.